### PR TITLE
#138 Add decorative Tombstone

### PR DESCRIPTION
**Pull request by vronc**
I made a tombstone and it seems to be working!

The only thing that I can't get to work is rotating it and getting the actual bounding box voxel (the collider) to follow. I tried to make it an entity and mimic the spooky chest that I saw had code for being positioned differently according to the player's position when placing it. But not much success for me on that part! :confounded: Any tips for getting it to rotate?

### CODE REVIEW

**Comment by Shnupbups** [comment]
To get it to rotate you're gonna want it to have a direction blockstate property that is set based on the player's direction when placing the block. Then you'll need to return a different VoxelShape from `getOutlineShape` based on the blockstate, and make sure the blockstate json rotates the model too.

**Comment by vronc** [comment]
Thank you for your help! I've solved it now.
I got one more problem that is probably pretty easy to solve but I can't find the solution; when I place a big block next to the tombstone the side of the tombstone isn't rendered. I changed rendering type to cutout but it still doesn't work for adjacent 16x16 blocks.

**Comment by vini2003** [comment]
I'll take a look.

**Comment by Juuxel** [comment]
That sounds like an issue with the model's cullfaces.

**Comment by vini2003** [comment]
Yep, just confirmed. You don't need to specify `cullface` in the model JSON.

**Comment by vronc** [comment]
Ah, you're being so helpful!

I would say I'm "done". Would someone review? 😇

**Comment by Juuxel** [comment]
Does the model have a bottom face? (for when it's on top of a transparent block or floating)

**Comment by vronc** [comment]
Oh! Now it does.

**Comment by vini2003** [comment]
Fix this minor conflict and we can merge.

**Comment by vronc** [comment]
Done!

File: src/main/java/com/fabriccommunity/spookytime/block/TombstoneBlock.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+		)
+		.collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
+
+	public TombstoneBlock(Settings settings) {
+		super(settings);
+	}
+
+	@Override
+	public BlockState getPlacementState(ItemPlacementContext placementContext) {
+
```
Comment (Shrimpstress) [inline src/main/java/com/fabriccommunity/spookytime/block/TombstoneBlock.java]: Remove this whitespace.

**Review by Shrimpstress — APPROVED** [review]
This gets an approval from me, but I'd like to see this whitespace removed before I merge.

**Pull request merged**